metrics-operator/Dockerfile

```diff
@@ -8,7 +8,7 @@ COPY go.mod go.sum ./
 RUN go mod download
 
 # renovate: datasource=github-releases depName=kubernetes-sigs/controller-tools
-ARG CONTROLLER_TOOLS_VERSION=v0.16.4
+ARG CONTROLLER_TOOLS_VERSION=v0.16.5
```

Comment:
can we bump this for metrics-operator as well @odubajDT ?

Reply:
yes, should be ok